src/test/calculate.test.js

```diff
@@ -0,0 +1,49 @@
+import Calculate from '../logic/calculate';
```

Comment:
- [ ] It seems the modifier operator `%` is not being tested. We need to implement at least a test for it.